lib/ui/article/ArticleList.dart

```diff
@@ -1,59 +1,57 @@
 import 'package:farmsmart_flutter/model/bloc/ViewModelProvider.dart';
-import 'package:farmsmart_flutter/ui/common/ViewModelProviderBuilder.dart';
-import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
 import 'package:farmsmart_flutter/ui/article/ArticleDetail.dart';
-import 'package:farmsmart_flutter/ui/article/viewModel/ArticleDetailViewModel.dart';
-import 'package:farmsmart_flutter/ui/article/viewModel/ArticleListItemViewModel.dart';
 import 'package:farmsmart_flutter/ui/article/HeroListItem.dart';
 import 'package:farmsmart_flutter/ui/article/StandardListItem.dart';
-import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/ui/article/viewModel/ArticleDetailViewModel.dart';
+import 'package:farmsmart_flutter/ui/article/viewModel/ArticleListItemViewModel.dart';
 import 'package:farmsmart_flutter/ui/article/viewModel/ArticleListViewModel.dart';
+import 'package:farmsmart_flutter/ui/common/ViewModelProviderBuilder.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+import 'package:flutter/material.dart';
 
-abstract class ArticleListStyle {
+class ArticleListStyle {
```

Comment:
I've refactored this to the new way because I needed it to create 2 styles (discover & community)

Reply:
👍